packages/app/src/cli/services/dev/processes/dev-session.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable no-case-declarations */
+import {BaseProcess, DevProcessFunction} from './types.js'
+import {installJavy} from '../../function/build.js'
+import {DeveloperPlatformClient} from '../../../utilities/developer-platform-client.js'
+import {AppInterface} from '../../../models/app/app.js'
+import {getExtensionUploadURL} from '../../deploy/upload.js'
+import {AppEventWatcher, EventType} from '../app-events/app-event-watcher.js'
+import {performActionWithRetryAfterRecovery} from '@shopify/cli-kit/common/retry'
+import {mkdir, readFileSync, rmdir, tempDirectory, writeFile} from '@shopify/cli-kit/node/fs'
+import {dirname, joinPath} from '@shopify/cli-kit/node/path'
+import {AbortSignal} from '@shopify/cli-kit/node/abort'
+import {zip} from '@shopify/cli-kit/node/archiver'
+import {formData} from '@shopify/cli-kit/node/http'
+import {outputDebug, outputWarn} from '@shopify/cli-kit/node/output'
+import {endHRTimeInMs, startHRTime} from '@shopify/cli-kit/node/hrtime'
+import {Writable} from 'stream'
+
+interface DevSessionOptions {
+  developerPlatformClient: DeveloperPlatformClient
+  storeFqdn: string
+  apiKey: string
+  url: string
+  app: AppInterface
+  organizationId: string
+  appId: string
+}
+
+interface DevSessionProcessOptions extends DevSessionOptions {
+  bundlePath: string
+  stdout: Writable
+  stderr: Writable
+  signal: AbortSignal
+}
+
+export interface DevSessionProcess extends BaseProcess<DevSessionOptions> {
+  type: 'dev-session'
```

Comment:
We call it `dev session` here, but `consistent dev` in other places, could we also be consistent? 😜

Reply:
I've updated it to dev sessions everywhere